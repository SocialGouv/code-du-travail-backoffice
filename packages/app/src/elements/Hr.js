import styled from "@emotion/styled";

export default styled.hr`
  border: 0;
  border-top: solid 1px var(--color-border);
  margin: 1rem 0;
`;
