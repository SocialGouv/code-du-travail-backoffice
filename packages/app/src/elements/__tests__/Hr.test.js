import React from "react";

import Hr from "../Hr";

describe("elements/<Hr />", () => {
  it(`should pass`, () => {
    const $hr = testRender(<Hr />);

    expect($hr).toMatchSnapshot();
  });
});
