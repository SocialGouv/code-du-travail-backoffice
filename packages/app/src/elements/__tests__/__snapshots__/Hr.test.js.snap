// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elements/<Hr /> should pass 1`] = `
.emotion-0 {
  border: 0;
  border-top: solid 1px var(--color-border);
  margin: 1rem 0;
}

<hr
  className="emotion-0 emotion-1"
/>
`;
